src/io/test.zig

```diff
@@ -99,9 +106,34 @@ test "write/read/close" {
             completion: *IO.Completion,
             result: IO.CloseError!void,
         ) void {
-            _ = completion;
             _ = result catch @panic("close error");
 
+            if (builtin.target.os.tag == .linux) {
+                self.io.statx(
+                    *Context,
+                    self,
+                    statx_callback,
+                    completion,
+                    os.AT.FDCWD,
+                    self.path,
+                    0,
+                    os.linux.STATX_BASIC_STATS,
+                    &self.statx,
+                );
+            } else {
+                self.done = true;
```

Comment:
Maybe same trick again?
```suggestion
                self.statx_callback(completion, {});
```

Reply:
Ah yes

Reply:
Ah no - Darwin / Win don't have the IO.StatxError type.